app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    cp, file1, file2 = command.split()
+    if cp != "cp":
+        return
+    if file1 == file2:
+        return
+
+    with open(file1, "r") as f1, open(file2, "w") as f2:
```

Comment:
Code Style: You're using single-character variable names (f1 and f2) for the source file and the destination file. This is not in line with the best practices for variable naming. Please use more descriptive names, like 'source_file' and 'destination_file'.